### PR TITLE
Revtui improvements

### PR DESCRIPTION
**Pull request by grisu48**
Three improvements for `openqa-revtui`:

* `reviewed` state now includes all failure states
* Improved examples (hide running and softfailed jobs)
* On startup error, the error message is displayed

### CODE REVIEW

**Comment by grisu48** [comment]
Before merging, I want to test the changes in tomorrow's review round.

File: _review/containers.toml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-RabbitMQ = "amqps://suse:suse@rabbit.suse.de"    # RabbitMQ instance to query
-RabbitMQTopic = "suse.openqa.job.done"           # RabbitMQ topic to query
-HideStatus = [ "scheduled", "passed" ]           # Hide scheduled and passed jobs
-RefreshInterval = 60                             # Refresh from API once every minute
-MaxJobs = 20                                     # Max. job per group to display
-GroupBy = "groups"                               # Group by defined groups ("none" or "groups")
+Instance = "https://openqa.suse.de"                                    # openQA instance to query
+RabbitMQ = "amqps://suse:suse@rabbit.suse.de"                          # RabbitMQ instance to query
+RabbitMQTopic = "suse.openqa.job.done"                                 # RabbitMQ topic to query
+HideStatus = [ "scheduled","passed","running","softfailed" ]           # Hide scheduled and passed jobs
```
Comment (d3flex) [inline _review/containers.toml]: add `assigned` as well.

File: cmd/openqa-revtui/openqa-revtui.go [hunk 2]
```diff
@@ -414,9 +414,12 @@ func main() {
 		os.Exit(1)
 	}
 	tui.SetHideStatus(cf.HideStatus)
-	rc := tui_main(&tui, instance)
+	err := tui_main(&tui, instance)
```
Comment (d3flex) [inline cmd/openqa-revtui/openqa-revtui.go:417]: not sure if this is a better name for this var

Comment (grisu48) [reply cmd/openqa-revtui/openqa-revtui.go:417]: The type of the variable changed as well. Before it was a return code (`int`), now it is an `error` type.

File: _review/containers.toml (hunk 1 above)
Comment (grisu48) [reply _review/containers.toml]: Good catch!

**Pull request merged**